.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     steps:
       - attach_workspace: 
           at: .
-      - run: npx eslint .
+      - run: npm run lint
```

Comment:
Why did this have to change?

Reply:
There was no need, I just decided to introduce an npm script for linting. Will revert it.